gallery/package.json

```
@@ -0,0 +1,44 @@
+{
+    "name": "gallery",
+    "license": "Apache-2.0",
+    "version": "0.1.",
+    "packageManager": "yarn@3.2.0",
+    "dependencies": {
+        "@concordium/browser-wallet-api-helpers": "^2.0.0",
+        "@concordium/web-sdk": "^3.1.0-beta",
+        "react": "^18.1.0",
+        "react-dom": "^18.1.0"
+    },
+    "devDependencies": {
+        "@craftamap/esbuild-plugin-html": "^0.4.0",
+        "@types/react": "^18.0.9",
+        "@types/react-dom": "^18.0.5",
+        "eslint": "7",
+        "eslint-config-prettier": "^8.5.0",
+        "eslint-plugin-import": "^2.26.0",
+        "eslint-plugin-jsx-a11y": "^6.5.1",
+        "eslint-plugin-prettier": "^4.0.0",
+        "eslint-plugin-react": "^7.29.4",
+        "eslint-plugin-react-hooks": "^4.4.0",
+        "cross-env": "^7.0.3",
+        "esbuild": "^0.14.42",
+        "esbuild-plugin-svgr": "^1.0.1",
+        "live-server": "^1.2.2",
```

```suggestion
        "live-server": "1.2.1",
```

I had to `yarn add -D live-server@1.2.1` to be able to serve the app.. There is a bug with line endings in v1.2.2 apparently:
https://github.com/tapio/live-server/issues/394